modeline/disk/README.org

```diff
@@ -6,14 +6,16 @@ This module prints some information about your disks on the modeline.
 
 Put:
 #+BEGIN_SRC common-lisp
-    (ql:quickload "cl-diskspace")
+  (ql:quickload "cl-diskspace")
+  ;; on GNU/linux only
+  (ql:quickload "cl-mount-info")
```

Comment:
:scissors: This is not needed as cl-diskspace declares cl-mount-info as a dependency

Reply:
uhm... i can not find any mention of `cl-mount-info` in the asdf file of `cl-diskspace`, are you sure it is a dependency?

Reply:
You are right I got confused and thought the plugin was cl-diskspace instead of disk. Quickloading the disk system should be enough for the first time but the instructions are also good as is (as after fetching the systems once asdf will find them on the next time they are loaded.